client/pages/episode-list.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+class EpisodeList extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { episodes: this.props.show };
+  }
```

Comment:
You can remove this constructor, if you need to filter your episodes, you can get it at `this.props.episodes`.